application/common/sync/Synchronizer.php

```diff
@@ -137,22 +137,32 @@ protected function activateAndUpdateUser(User $user)
      */
     protected function activateAndUpdateUsers(array $usersToUpdateAndActivate)
     {
+        $usersWithErrors = [];
+        $lastError = null;
+
         $employeeIdsOfUsersUpdated = [];
-        foreach ($usersToUpdateAndActivate as $userToUpdateAndActivate) {
+        foreach ($usersToUpdateAndActivate as $nextUser) {
```

Comment:
Why the variable name change? When I read "next user", logic tells me it's the one AFTER the current one.

Reply:
Because `$userToUpdateAndActivate` is too long and looks almost exactly like `$usersToUpdateAndActivate`.
Also, it's fairly obvious what the short version is in the context, especially when the `for` loop is only a few lines long.